frontend/src/components/ Footer/footer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import logo from './imagenes/logo.png';
+import redes from './imagenes/redes.png';
+import binamp from './imagenes/logobinamp.png';
+import trazado from './imagenes/Trazado.png';
+import copyryght from './imagenes/copyryght.png';
+
+class Fooster extends React.Component {
+ render(){
+    return(
+        <footer>    
+     <div className={styles.containerbody}>
+      <div className={styles.column}>
+         <img  className={styles.imagen1} src={logo}  alt="logo"/>      
+      </div>  
+           <div className={styles.row1}>
+            <h1>Contact US</h1>
+               <label>Kirtlington Park Polo Club</label>
+                  <label>akerman Street</label>
+                    <label> Kirtlington</label>
+                      <label>OXS 3JQ</label>
+                   </div>        
+         <img src={trazado} alt="trazado"  className={styles.trazado}/>
+              <div className={styles.row2}>
+                <img className={styles.imagen2} src= {redes} alt="redes" />
+               <label   style={{ width: "160px" }}>+44 (0) 1865 600 448</label>
+                 <label style={{ width: "160px" }}>info@mendardpolo.com</label> 
+              </div>
```

Comment:
Un part de detallitos acá, pero más que nada para mantener la consistencia:
- Tener en cuenta los espaciados
- Tener en cuenta la identación
```suggestion
  <div className={styles.row2}>
    <img className={styles.imagen2} src={redes} alt="redes" />
    <label style={{ width: "160px" }}>+44 (0) 1865 600 448</label>
    <label style={{ width: "160px" }}>info@mendardpolo.com</label> 
  </div>
```